src/pybind/mgr/dashboard/cherrypy_backports.py

```diff
@@ -174,7 +174,7 @@ def uuid4(self):
                 It's evaluated lazily on render.
                 """
                 try:
-                    self._uuid4
+                    self._uuid4  # type: ignore
```

Comment:
Don't understand why `# type: uuid.UUID` does not make mypy happy.

Reply:
which error do you get?

Reply:
```
dashboard/module.py:30: note: In module imported here:
dashboard/cherrypy_backports.py: note: In member "uuid4" of class "LazyUUID4":
dashboard/cherrypy_backports.py:177: error: Cannot determine type of '_uuid4'
Found 1 error in 1 file (checked 9 source files)
```